packages/snjs/test/factory.ts

```diff
@@ -78,49 +116,70 @@ export function getDefaultWebSocketUrl() {
   return 'ws://localhost';
 }
 
-export function createApplication(identifier, environment, platform) {
-  const deviceInterface = new WebDeviceInterface(
+const getSwappedClasses = (environment?: Environment) => {
+  const classMap = {
+    swap: SNComponentManager,
+    with: WebComponentManager
+  };
+  switch (environment) {
+    case Environment.Mobile:
+      classMap.with = MobileComponentManager;
+      break;
+    default:
+      classMap.with = WebComponentManager;
+      break;
+  }
+  return [classMap];
+};
+
+export function createApplication(identifier: string, environment?: Environment, platform?: Platform) {
+  /**
+   * Stops loading any resources or network requests.
```

Comment:
Why?

Reply:
@johnny243 can you add a comment stating why in the code? This type of knowledge will be lost really quick without context.

Reply:
Done 👍🏽